api/src/test/java/edu/cornell/mannlib/vitro/webapp/dynapi/SolrQueryTest.java

```diff
@@ -0,0 +1,96 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi;
+
+import static org.easymock.EasyMock.expect;
```

Comment:
I would like to bring up discussion on whether we should be using easy mock at all. Before work @kaladay did it was only used on a single line of tests where it wasn't even required. I am not sure it is a good testing framework. It also deviates from all other testing strategy in Vitro. The question then is, do we want to begin defining a new testing strategy? If so, should we use easymock? If neither, it is doable to continue using Vitro testing strategies.

Reply:
I'm not that familiar with different testing strategies and different frameworks, so I can't really add to this discussion. Basically tried to emulate tests used in RPCEndpointTest :) Not sure how to mock all of the components SolrOperation uses, such as Application and SearchEngine without a framework for mocking. Should I create instances of those classes manually before each test scenario? Thanks for detailed comments

Reply:
Yeah, we can discuss. Maybe we continue to use easymock.

Reply:
We have decided to remove the dependency easymock and use mockito. Is much more intuitive and easier to use. Can find examples in new refactored RPCEndpointTest. Please update to use mockito. https://github.com/vivo-project/Vitro/blob/sprint-dynapi-2022-feb-staging/api/src/test/java/edu/cornell/mannlib/vitro/webapp/dynapi/RPCEndpointTest.java